SR2021/2021-01-01-january-update.md

```diff
@@ -13,6 +13,17 @@ The first league session will be happening on the 16th of January, at 12:00 UK t
 
 Further details, and a link to the livestream, will be on our website soon - for this and the other league sessions.
 
+## Simulator Update
+
+We released a new version of the simulator, which adds a few important improvements:
+
+- Support for Webots 2021a (no more deprecation warnings).
+- Add several scientific libraries. See [the docs](https://studentrobotics.org/docs/programming/python/libraries) for a full list.
+- Reconstructed arena to fix issues with distance sensors and shadows.
```

Comment:
I think we should explicitly call out here that as a result of this rebuild the positions of the internal walls have changed slightly. It's only a couple of centimetres at most, but if someone is dead-reckoning it might throw them off.

Reply:
92374ce726dac21c2b998af2eaafaa75dbabfb53